libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/useShareDialogBase.ts

```diff
@@ -153,12 +155,18 @@ export const useShareDialogBase = (props: IShareDialogBaseProps): IUseShareDialo
         onAddGranteeBackClick,
         onLockChange,
         onUnderLenientControlChange,
+        onGranularGranteeChange,
     } = useShareDialogState(isUnderLenientControl, isLocked);
 
     const onLoadGranteesSuccess = useCallback(
         (result: GranteeItem[]) => {
-            const groupAll = mapShareStatusToGroupAll(shareStatus);
-            onLoadGrantees(result, groupAll);
+            // Don't load "All group" for Panther
```

Comment:
This comment is not necessary. The condition is self explanatory IMHO. Also, it could become obsolete when new backend will be added and you will just define capabilities for it and expect it will work as expected. You would not go through the code to check every capability to see if there is some mention of backend name :)

Reply:
Removed.